airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/configuration.py

```diff
@@ -35,9 +35,11 @@ class FormatConfiguration:
     # Run prettier on all json and yaml files.
     FormatConfiguration(
         Formatter.JS,
-        ["**/*.json", "**/*.yaml", "**/*.yml"],
+        ["**/*.json", "**/*.yaml", "**/*.yml", "**/*.md"],
         format_js_container,
-        [f"prettier --write . --list-different --cache --cache-location={CACHE_MOUNT_PATH}/.prettier_cache"],
+        [
```

Comment:
I still get a mismatch locally, when I run airbyte-ci format fix js, it does NOT see those files that are in this PR. I had to run prettier locally instead. I don't know what's borked in my local dagger, but I'm very curious. Not a high-leverage work item anymore. /cc @alafanechere

Reply:
It'd be interesting to set a breakpoint [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/format_command.py#L87) to double check if markdown files are mounted to the `dir_to_format`.
Did you run the command from the root of the repo?
Did you check that [`DEFAULT_FORMAT_IGNORE_LIST`](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/consts.py#L12) does not override your filters?